bucket/nuget-package-explorer.json

```diff
@@ -3,15 +3,19 @@
     "description": "a GUI tool for creating, updating, and deploying NuGet packages",
     "version": "5.0.230",
     "license": "MIT",
-    "url": "https://github.com/NuGetPackageExplorer/NuGetPackageExplorer/releases/download/v5.0.230/PackageExplorer.5.0.230.zip",
-    "hash": "2151f830c7a62ad41169d23037ffb7051b53aba8e612d338fc6f22e5a77046d2",
+    "url": "https://packages.chocolatey.org/NugetPackageExplorer.5.0.230.nupkg",
+    "hash": "4df8ece75be4fef611f2fb8bd446e7f4d22c60b01b92845d446366369dcb2238",
     "checkver": {
-        "github": "https://github.com/NuGetPackageExplorer/NuGetPackageExplorer"
+        "url": "https://chocolatey.org/packages/NugetPackageExplorer",
+        "regex": "Chocolatey.*?([\\d.]+)"
     },
     "autoupdate": {
-        "url": "https://github.com/NuGetPackageExplorer/NuGetPackageExplorer/releases/download/v$version/PackageExplorer.$version.zip"
+        "url": "https://packages.chocolatey.org/NugetPackageExplorer.$version.nupkg",
+        "hash": {
+            "url": "https://chocolatey.org/packages/NugetPackageExplorer",
+            "regex": "$sha256.*?$basename"
+        }
     },
-    "bin": "nugetpackageexplorer.exe",
```

Comment:
Why did you remove the `bin` property?

Reply:
Sorry, I think Nuget Package Explorer is a pure GUI app, but it could open `*.nupkg` through command, so add it again.